packages/cronjob-rss-feed/configs.js

```diff
@@ -0,0 +1,14 @@
+export const config = {
+  apiUrl: 'https://kids-cms.twreporter.org/api/graphql',
+  baseUrl: 'https://kids.twreporter.org/',
```

Comment:
Question:
Could we use environment variables to assign `apiUrl`, `baseUrl` and `bucketName`?
We might need dev or staging environment to test.

Reply:
This code is outdated